DER/DerivativesContracts/ExoticOptions.rdf

```diff
@@ -663,7 +655,7 @@ Further notes: REVIEW THIS - there appear to be two separate semantics in this l
 		<skos:definition xml:lang="en">The price or other variable (as a number) which has to be met for the event to be deemed to have occurred.</skos:definition>
 	</owl:DatatypeProperty>
 	
-	<owl:Class rdf:about="&fibo-der-der-opt;OptionPremium">
+	<owl:Class rdf:about="&fibo-der-drc-opt;OptionPremium">
```

Comment:
@rivettp That was something "leftover" that was defined in this exotic options ontology so rather than remove it I left it in for discussion (and likely eventual removal)